README.md

```diff
@@ -135,9 +135,6 @@ You may want to combine data from other sources with your IB data.  [Deephaven](
 * [Kafka](https://deephaven.io/core/docs/how-to-guides/kafka-topics/).  
 See the [Deephaven Documentation](https://deephaven.io/core/docs) for details.
 
-Files placed in the `./docker/data/` directory are visible in the Docker container at `/data/`.  
-See [Access your file system with Docker data volumes](https://deephaven.io/core/docs/conceptual/docker-data-volumes/) for details.
-
 # Run deephaven-ib
 
 Follow these steps to run a [Deephaven](https://deephaven.io) plus [Interactive Brokers](https://interactivebrokers.com) system. 
```

Comment:
I can't comment on line 147 directly because it's not in the diff. It seems like everything is now via pip-installed DH. In that case, line 147 directs me to the quickstart. If I blindly follow the Docker install commands, will things still work? Should I be specifically directed to the pip-install section? What about the advanced pip-install doc?

Reply:
Link to the quick start was removed.  The install script does it all.